pkg/doc/signature/ed25519signature2018/suite_test.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ed25519signature2018
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"crypto/ed25519"
+)
+
+func TestSignatureSuite_Sign(t *testing.T) {
+	_, privKey, err := ed25519.GenerateKey(rand.Reader)
+	require.NoError(t, err)
+	require.NotEmpty(t, privKey)
+
+	ss := New()
+	signature, err := ss.Sign(privKey, []byte("test doc"))
+	require.NoError(t, err)
+	require.NotEmpty(t, signature)
+}
+
+func TestSignatureSuite_Verify(t *testing.T) {
```

Comment:
Do you have any example documents with signatures that you can use to test Sign & Verify?
If there is test data elsewhere (provided by the spec? In the tests of reference implementations? Generated by reference implementations?) then you can use such data to test that Sign produces the right signature, and to test that Verify will accept data with correct signatures and reject data with incorrect signatures.

Reply:
You can also test that canonicalization will produce the same result even if you change whitespace, element order etc in a document.

Reply:
We are using standard go library "crypto/ed25519" to produce signature and verify. I don't think that we need additional unit tests to verify this.

Reply:
Canonicalization will aways produce the same result (same order of elements) and my unit test already tests that. I chose Test 28 from https://json-ld.org/test-suite/reports/#test_30bc80ba056257df8a196e8f65c097fc. 

json-gold (library used) passed all 119 tests